example/ipsec_api/odp_ipsec_api_run_router.sh

```diff
@@ -3,7 +3,55 @@
 # Live router test
 #  - 2 interfaces interfaces
 #  - Specify API mode on command line
-sudo ./odp_ipsec_api -i p7p1,p8p1 \
--r 192.168.111.2/32,p7p1,08:00:27:76:B5:E0 \
--r 192.168.222.2/32,p8p1,08:00:27:F5:8B:DB \
--c 1 "$@"
+
+# IPSEC_APP_MODE: 0 - STAND ALONE, 1 - LIVE, 2 - ROUTER
+IPSEC_APP_MODE=2
+
+if  [ -f ./pktio_env ]; then
+	. ./pktio_env
+else
+	echo "BUG: unable to find pktio_env!"
+	echo "pktio_env has to be in current directory"
+	exit 1
+fi
+
+setup_interfaces
+
+# this just turns off output buffering so that you still get periodic
+# output while piping to tee, as long as stdbuf is available.
+STDBUF="`which stdbuf 2>/dev/null` -o 0" || STDBUF=
+LOG=odp_ipsec_api_tmp.log
+PID=app_pid
+
+($STDBUF \
+ ./odp_ipsec_api -i $IF0,$IF2 \
+	-r 192.168.111.2/32,$IF0,$MAC_ADDR_IF1 \
+	-r 192.168.222.2/32,$IF2,$MAC_ADDR_IF3 \
```

Comment:
Same comments as for `example/ipsec_api/odp_ipsec_api_run_live.sh`.

Reply:
Ack.